payroll-processor-client/src/app/payroll/payroll-charts/payroll-charts.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="container">
+  <div class="row">
+    <div class="col-6">
+      <h6>Employee Payroll Distribution</h6>
+      <ngx-charts-pie-chart
+        [view]="view"
```

Comment:
There's not much getting around these large list of properties for highly configurable components.

There are ways around it, but they're not worth the abstraction atm.